src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTests/IotHubTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.ServiceManagemenet.Common.Models;
+using Microsoft.Azure.Test;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Commands.IotHub.Test.ScenarioTests
+{
+    public class IotHubTests : RMTestBase
+    {
+        public IotHubTests(ITestOutputHelper output)
+        {
+            XunitTracingInterceptor.AddToContext(new XunitTracingInterceptor(output));
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestAzureIotHubLifeCycle()
+        {
+            IotHubController.NewInstance.RunPsTestWorkflow(
+                () => { return new[] { string.Format("{0} {1} {2} {3} {4}", "Test-AzureRmIotHubLifecycle", "northeurope", "powershelliothub", "powershellrg", "S1") }; },
```

Comment:
Found other resources hardcoding location directly in the script. What I have done is to ensure that the script itself does not have the hardcoding so it can be run in any location. Let me know if this works for the current PR. I will raise an issue to fix this in a subsequent PR.